### ISSUE TITLE
Chinese Translation Issue

### ISSUE THREAD

**Issue by neomonkeus** [niftools/nifskope#19]
A user on sf.net - icedream has provided a chinese translation.

> I've done a Chinese translation.
> 
> Problems are some icons doesn't show correctly if I've translated the strings, this is the screenshot: 
> 
> ![txt_issue](https://f.cloud.github.com/assets/525919/1639408/912d9f7c-581f-11e3-9bd5-04733bb2f849.png)
> 
> Another problem is that "Continue" has two meanings in spells/blocks.cpp, one for continue/cancel, one for Collision Detection, but I can't separate them for two meanings.
> 
> Also, some strings should have been translable but not yet, such as the menu "Spells", redo/redo in UV editor.
- [ ] Some icons don't show correctly if using translated strings, see screenshot: 
- [ ] "Continue" has two meanings in spells/blocks.cpp, one for continue/cancel, one for Collision Detection, but I can't separate them for two meanings.
- [ ] Missing strings that should have been translatable but not yet implemented, such as the menu "Spells", redo/redo in UV editor.

**Comment by neomonkeus** [comment]
Initially getting this integrated should be pretty trivial, probably do this myself to get into NifSkope coding. 
I will add sub-tasks for other issues mentioned as I suspect they are across other translations too.

**Comment by hexabits** [comment]
Yeah, I am not worried about multi-language support myself.  In fact I may need to get other people to make appropriate translations for any UI changes I make. 

Also, I'll have to ponder on it more but I'm not sure if that "Txt" area is the best way to deal with the strings.  That'll be a later enhancement.  If it used iconography instead, something that surpasses language boundaries, it may be better.

**Comment by neomonkeus** [comment]
I have recruited several minions who are currently looking into an initial validation & verification of existing translations.

I like the idea of iconography, just add an issue. They can be associated with milestones further down the line.

**Comment by skyfox69** [comment]
If you haven't found a minion for German translation yet, I could have a look. Hmmm, with a size of 2.01m I would be a rather tall minion ;-)

**Comment by neomonkeus** [comment]
@skyfox69 had Tamira do an initial check and she is asking about on some forums for input, so any additional input can only help. 

@l0fka has created an initial Russian translation so will get a pull request once I sort compiling issues.

**Comment by L0FKA** [comment]
Hey, @neomonkeus, I want you generate Default.ts from new nifskope, not all strings i can translate now..

**Comment by neomonkeus** [comment]
Uploaded files from sf.net - https://sourceforge.net/projects/skyrimtoolschs/

**Comment by neomonkeus** [comment]
Extracted out functional problem relation to the translation process.
